app/models/error_report.rb

```diff
@@ -67,11 +67,27 @@ def error
       fingerprint: fingerprint
     )
   end
+  delegate :problem, to: :error
 
   def valid?
     !!app
   end
 
+  def should_keep?
+    # TODO: The main version of Errbit has introduced `current_app_version` to
+    # its App model and uses it here to ensure it doesn't field newer rerquests
+    # than what it's able to understand. That might be nice to bring over down
+    # the line, but at the moment there's no need to introduce the concept and
+    # migrate our current data just to support the new Airbrake API
+
+    # app_version = server_environment["app-version"] || ""
+    # current_version = app.current_app_version
+    # return true unless current_version.present?
+    # return false if app_version.length <= 0
+    # Gem::Version.new(app_version) >= Gem::Version.new(current_version)
+    true
+  end
+
```

Comment:
For the sake of this PR, let's just drop `should_keep?`, line 21 from the `notices_controller.rb` and the constant `VERSION_TOO_OLD` — it's all unrelated to supporting the Airbrake API v3.

Reply:
👌